tests/pytests/test_mdi.py

```diff
@@ -0,0 +1,141 @@
+import pytest
+from qcelemental.testing import compare, compare_recursive, compare_values, tnm
+
+import psi4
+
+
```

Comment:
find the marker for "quick" tests and add it here, so it at least tries to run. is mdi ready as a conda package so we can test it in CI?

Reply:
Before or by the time MDI is available as a conda package, it'd be good to add a test to test_addons.py, too, as those are the "smoke" tests for the psi4 interfaces.